### PR TITLE
[WIP] Add chart for selenium grid

### PR DESCRIPTION
**Pull request by yuvipanda**
Sets up an installation of https://github.com/SeleniumHQ/selenium/wiki/Grid2

Slightly stolen from @sgoings's https://github.com/helm/charts/tree/master/selenium-hub

### CODE REVIEW

File: stable/selenium-grid/values.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  # Set to true if the hub service should be exposed
+  # to the internet with a LoadBalancer
+  expose: false
+  # Image and tag of the selenium hub to use.
+  image:
+    name: selenium/hub
+    tag: 3.0.1
+  resources:
+    requests:
+      cpu: 2
```
Comment (lachie83) [inline stable/selenium-grid/values.yaml:11]: Can you please confirm if these are the lowest resource requirements possible?

Comment (yuvipanda) [reply stable/selenium-grid/values.yaml:11]: I needed these when I ran a 100 count hub but can lower these since the default is just 2.

**Review by lachie83 — CHANGES_REQUESTED** [review]
Can you please add a README? Also please double check that you've me all the tech requirements for this to be merged? 
https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements

**Comment by flah00** [comment]
@yuvipanda what do you think of #665? There are a number of options which this chart doesn't include, which are pretty critical to my environment

* ability to use debug nodes
* ability to turn on and off nodes, which aren't needed
* ability to set `JAVA_OPTS` and `SE_OPTS`
* ability to set screen size, tz, etc
* support for chrome shared memory above docker default of 64m

**Comment by yuvipanda** [comment]
@flah00 w00t! I like that one much better! You seem to actually know things about selenium, while I was just trying to get a minimal setup up. I'll close this one in favor of yours :)

**Pull request closed without merging**